content/pages/index.html

```diff
@@ -66,6 +66,7 @@ <h2>Self hostable</h2>
 				<td>
 					<p>Required: <a href="http://php.net/xml">libxml</a>, <a href="http://php.net/curl">cURL</a>, <a href="http://php.net/dom">DOM</a> <a href="http://php.net/pdo-mysql">PDO_MySQL</a></p>
 					<p>Recommended: <a href="http://php.net/json">JSON</a>, <a href="http://php.net/gmp">GMP</a>, <a href="http://php.net/zlib">Zlib</a>, <a href="http://php.net/intl.idn">IDN</a>, <a href="http://php.net/mbstring">mbstring</a>, <a href="http://php.net/iconv">iconv</a>, <a href="http://php.net/zip">Zip</a></p>
+					<p>For whole modules list see <a href="https://github.com/FreshRSS/FreshRSS/blob/744a9e8cf00aef7dec0acfa5f90f0dcfa2ef8837/Docker/Dockerfile-Alpine#L7-L9">Dockerfile</a></php>
```

Comment:
Typo :-) `</p>`and not `</php>`

Reply:
Oopsie

How can I fix it? (a new PR ? :s )

Reply:
Just push new changes to your same branch - I can see you found out :-)

Reply:
Done for me.

Thx :)